.github/workflows/ci.yml

```diff
@@ -533,7 +533,7 @@ jobs:
       if: contains(github.event.pull_request.labels.*.name, 'build-proxy-image')
       run: ./build.sh -i proxy
     - name: Build agent
-      run: SYSTEM_TEST_BUILD_ATTEMPTS=3 ./build.sh java_otel -i agent
+      run: SYSTEM_TEST_BUILD_ATTEMPTS=3 AGENT_BASE_IMAGE="datadog/agent:7.48.0-rc.3" ./build.sh java_otel -i agent
```

Comment:
Just for my understanding, why hard coding this version ?

Reply:
The new feature that we want to test is only available in 7.48. If I don't specify the `rc.3` version, Docker will pull 7.47 by default.

Reply:
Great, thanks!

Reply:
-> #1577

Reply:
Merged. You can update your branch, remove the hardcoded agent version in the CI, and then use `@released(agent="1.2")` on test classes, `1.2` being the version where the feature is supposed to be ok.